src/autoscaler/api/brokerserver/broker_handler.go

```diff
@@ -104,6 +128,166 @@ func (h *BrokerHandler) CreateServiceInstance(w http.ResponseWriter, r *http.Req
 	}
 }
 
+func (h *BrokerHandler) UpdateServiceInstance(w http.ResponseWriter, r *http.Request, vars map[string]string) {
+	instanceId := vars["instanceId"]
+
+	body := &models.InstanceUpdateRequestBody{}
+	bodyBytes, err := ioutil.ReadAll(r.Body)
+	if err != nil {
+		h.logger.Error("failed to read service update request body", err, lager.Data{"instanceId": instanceId})
+		writeErrorResponse(w, http.StatusInternalServerError, "Failed to read request body")
+		return
+	}
+	err = json.Unmarshal(bodyBytes, body)
+	if err != nil {
+		h.logger.Error("failed to unmarshal service update body", err, lager.Data{"instanceId": instanceId, "body": string(bodyBytes)})
+		writeErrorResponse(w, http.StatusBadRequest, "Invalid request body format")
+		return
+	}
+
+	if instanceId == "" || body.ServiceID == "" {
+		h.logger.Error("failed to update service instance when trying to get mandatory data", nil, lager.Data{"instanceId": instanceId, "serviceId": body.ServiceID, "planId": body.PlanID})
+		writeErrorResponse(w, http.StatusBadRequest, "Malformed or missing mandatory data")
+		return
+	}
+
+	if body.Parameters == nil || body.Parameters.DefaultPolicy == nil {
+		h.logger.Error("failed to update instance, only default policy updates allowed", nil, lager.Data{"instanceId": instanceId, "serviceId": body.ServiceID, "planId": body.PlanID})
+		writeErrorResponse(w, http.StatusUnprocessableEntity, "Failed to update service instance: Only default_policy updates allowed")
+		return
+	}
```

Comment:
Would we use the "update" service instance to delete `default policy` by passing through `nil` parameters?  
i.e.  at first , I may "create-service -c xx" at first, and do some binding,  then I want to delete the default policy but don't want to unbind & delete the service instance.  Could I achieve it through "update" service instance?